server/src/webserver.rs

```diff
@@ -656,30 +663,69 @@ async fn serve_file(
         source_lexer(&file_contents, lexer)
     };
 
-	// <p>Convert doc blocks from Markdown to HTML then parse using HTML5Ever</p>
+    // <p>Convert doc blocks from Markdown to HTML then parse using HTML5Ever
+    // </p>
     let mut options = Options::empty();
     options.insert(Options::ENABLE_STRIKETHROUGH);
     for code_doc_block in &mut code_doc_block_arr {
         if let CodeDocBlock::DocBlock(ref mut doc_block) = code_doc_block {
             let parser = Parser::new_ext(&doc_block.contents, options);
             let mut html_output = String::new();
             html::push_html(&mut html_output, parser);
-		
-		// <p>Create a new RcDom object to represent the DOM tree with default settings</p>
-        let dom = RcDom::default();
-		// <p>Convert html_output to UTF-8 encoding then use parse_fragment to parse the HTML with default options</p>
-        let parse_result = html5ever::parse_fragment(dom,ParseOpts::default(),QualName::new(None, ns!(), "html".into()),Default::default(),).from_utf8().read_from(&mut html_output.as_bytes()).unwrap();
-		// <p>Create an empty vector for html_parsed_output</p>
-        let mut html_parsed_output = Vec::new();
-		// <p>Take the parsed html result document field, clone it, and create a serializable handle for it</p>
-        let serializable_handle: SerializableHandle = parse_result.document.clone().try_into().unwrap();
-		// <p>Serialize the handle and store in html_parsed_output</p>
-        serialize(&mut html_parsed_output, &serializable_handle, SerializeOpts::default()).unwrap();
-		// <p>Set the contents of the doc_block to the string converted UTF-8 encoded parsed HTML</p>
-        doc_block.contents = String::from_utf8(html_parsed_output).unwrap();
+
+            // <p>Create a new RcDom object to represent the DOM tree with
+            //     default settings</p>
+            let dom = RcDom::default();
+            // <p>Convert html_output to UTF-8 encoding then use parse_fragment
+            //     to parse the HTML with default options: parse_fragment is a
+            //     function of HTML5Ever that takes four arguments: (1) RcDom
```

Comment:
For you next PR: make this a numbered or bulleted list. Also add hyperlinks back to the relevant manual page (for `html5ever::parse_fragment`, etc.)